### PR TITLE
add model registry course link

### PR DESCRIPTION
**Pull request by kldarek**
## Description

We have a great model management course now - added the link to model registry guide intro page. 

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here: n/a

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [x ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ x] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ x] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

**Review by kldarek — COMMENTED** [review]
@ngrayluna I think the course recommendation belongs to the higher level bullet points, rather than "learn about" section? Conceptually, it's the biggest thing folks can take to advance their knowledge on model reg (compared to short videos, walkthrough or reports that we have above, and especially contrasting with specific features we list in "learn about" section. WDYT?

**Comment by kldarek** [comment]
@ngrayluna @noaleetz quick follow up on this, thanks!

File: docs/guides/model_registry/intro.md [hunk 1]
```diff
@@ -62,5 +62,6 @@ Depending on your use case, explore the following resources to get started with
    * [Protected models and access control](./access_controls.md).
    * [How to connect the Model Registry to CI/CD processes](./automation.md).
    * Set up [Slack notifications](./notifications.md) when a new model version is linked to a registered model.
+* Take the W&B online [Enterprise Model Management](https://www.wandb.courses/courses/enterprise-model-management) course.
```
Comment (noaleetz) [inline docs/guides/model_registry/intro.md]:
```suggestion
* Take the W&B [Enterprise Model Management](https://www.wandb.courses/courses/enterprise-model-management) course.   
```
@kldarek can you specify the key point(s) / areas that the course covers that relate to the Model Reg docs? i.e "take the w&b EMM course to learn about...

**Review by noaleetz — CHANGES_REQUESTED** [review]
added request for more specific CTA

File: docs/guides/model_registry/intro.md (hunk 1 above)
Comment (kldarek) [reply docs/guides/model_registry/intro.md]: Hey @noaleetz I think we cover Model Reg comprehensively in the course, do you think we should highlight anything in particular? I'd just say take the course to learn Model Registry best practices.

Comment (noaleetz) [reply docs/guides/model_registry/intro.md]: The CTA is not very convincing as is and how it ties into these docs. I agree that we cover Model Reg in the course, my ask though is that you specify a specifics on what the course covers in this CTA to connect it to the docs (i.e finish the sentence "take the course for an in-depth review on..."

Comment (kldarek) [reply docs/guides/model_registry/intro.md]: @noaleetz @ngrayluna How about this? 
Take the Weights & Biases Enterprise Model Management course and:
- learn how to use the W&B Model Registry to manage and version your models, track lineage, and promote models through different lifecycle stages.
- discover how to automate your model management workflows using webhooks and launch jobs.
- see how the Model Registry integrates with external ML systems and tools in your model development lifecycle for model evaluation, monitoring, and deployment.

Thanks to Agata for the copy!

Comment (noaleetz) [reply docs/guides/model_registry/intro.md]: sounds good to me - feel free to update the PR with these changes! Stamping

**Review by noaleetz — APPROVED** [review]

**Comment by kldarek** [comment]
@ngrayluna updated the PR as recommended by Noa above.

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**